command/watch.go

```diff
@@ -182,11 +185,19 @@ func (c *WatchCommand) Run(args []string) int {
 			cmd.Stdout = os.Stdout
 			cmd.Stderr = os.Stderr
 
+			// Watch for a stop
+			go func() {
+				<-wpStoppingCh
```

Comment:
That's correct. You're right, typically we use unbuffered channels and just `close()` them (which notifies all receivers), but it looks like for the `c.ShutdownCh` we send a struct down. Honestly that could probably be changed to a `close()`, but we can keep a separate channel here and fix that separately.